Online-Messaging-App/src/app/login/login.component.html

```diff
@@ -1,24 +1,30 @@
-<div class="container">
 
+<div class="container">
   <div *ngIf="common.checkIfLoggedIn(); else elseBlock">{{common.moveToHome()}}</div>
   <ng-template #elseBlock>
-    <p>You are not logged in. Please log in.</p>
-    <div class="row">
+  <app-header></app-header>
+    <div align="center" class="mat-display-2">Log in to Streamline Athletes Chat Service</div>
+    <div align = "center" class="row">
       <div class="col-lg-4">
         <form (ngSubmit)="loginSubmit(loginForm)" #loginForm="ngForm">
           <div class="form-group">
-            <label for="username">Username</label>
-            <input type="text" name="username" id="username" ngModel class="form-control">
+            <mat-form-field appearance="fill" color = "primary">
+              <mat-label>Username</mat-label>
+            <input matInput type="text" name="username" id="username" ngModel class="form-control">
```

Comment:
Minor thing, but this refers to the register component as well, the black bar that gets thicker when you hover over an input field for whatever reason gets longer and goes out of the box of the field. Can you fix it so it only gets thicker, but not longer